### ISSUE TITLE
getBridgeInterfacesVersion returns incorrect version of the contracts 

### ISSUE THREAD

**Issue by akolotov** [poanetwork/poa-bridge-contracts#92]
The Home Bridge contract for `refactor_v1` bridge (`erc-to-erc` mode) were deployed into Sokol network:
https://blockscout.com/poa/sokol/address/0xb5a5508af15042cbb7d529096f8ff589fb6f357f/transactions

Invocation of `getBridgeMode` method returns `0xba4690f5`. That's correct.

But when `getBridgeInterfacesVersion` is called, `0, 0, 0` is returned even if `BasicBridge.sol` has the following line:
```solidity
Version.Version public getBridgeInterfacesVersion = Version.Version(2, 0, 0);
```
The Remix Browser was used to call both methods:

![image](https://user-images.githubusercontent.com/20793260/46475360-fe27d500-c7ed-11e8-9c75-9c8502c90509.png)

**Comment by akolotov** [comment]
Yes, I think it makes sense to implement this in the suggested way. I would provide names for return values as so elaborate meaning of the digits:

```solidity
    function getBridgeInterfacesVersion() public pure returns(uint64 major, uint64 minor, uint64 patch) {
        return (2, 0, 0);
    }
```

**Comment by akolotov** [comment]
Addressed in #100
